### ISSUE TITLE
Uploading multiple files at once fails if an archive name is not specified

### ISSUE THREAD

**Issue by andreipoe** [timvisee/ffsend#109]
When trying to upload several files in one command, ffsend offers to archive them and upload the resulting archive. However, this will fail if `--name` is not specified:

```
$ ffsend upload file1 file2
You've selected multiple files, only a single file may be uploaded.
Archive the files into a single file? [Y/n]: y
Archiving...
error: you must specify a file name for the archive

Use '--name ' to specify a file name
For more information try '--help'
```

**Comment by timvisee** [comment]
This is intended behavior.

Or are you suggesting this can be improved?

**Comment by andreipoe** [comment]
Sorry if my original post wasn't very clear. Since it's asking a yes/no question, I would have expected the upload to proceed even if an archive name isn't given, by using an auto-generated name. 

If you think this is not desirable, then perhaps it's better to output the error message and stop before asking the question.

Thanks!

**Comment by timvisee** [comment]
I see. This is a side effect of how it is implemented.

I agree that generating a random name might be a better approach.

**Comment by timvisee** [comment]
`ffsend` now generates a random file name for unnamed archives (instead of showing an error). This is implemented in version `v0.2.59` and above.

This update should be ready for download once this release pipeline succeeds:  
https://gitlab.com/timvisee/ffsend/pipelines/131237744

**Comment by timvisee** [comment]
I had some trouble releasing this version as `snap` package. This has been done now. Just a heads up for if you were using this.
